src/script/event/NotificationService.ts

```diff
@@ -169,4 +168,33 @@ export class NotificationService {
       value: notificationId,
     });
   }
+
+  getNotificationIdByMessageId(messageId: string, clientId: string): Promise<string | void> {
+    return new Promise(async resolve => {
```

Comment:
Please do not mix `Promise` and `async` and use `for ... of` in line 187 instead.